File: app/src/main/java/com/datarockets/mnchkn/activities/dashboard/DashboardView.java
package com.datarockets.mnchkn.activities.dashboard;

import com.datarockets.mnchkn.models.Player;

import java.util.List;

public interface DashboardView {
    void finishGame();
    void setItems(List<Player> players);
    void showConfirmFinishGameDialog();
    void updatePlayerData(Player player, int position);
    void keepScreenOn(boolean keepActive);
}

File: app/src/main/kotlin/com/datarockets/mnchkn/activities/dashboard/DashboardView.kt
package com.datarockets.mnchkn.activities.dashboard

import com.datarockets.mnchkn.models.Player

interface DashboardView {
    fun finishGame()
    fun setItems(players: List<Player>)
    fun showConfirmFinishGameDialog()
    fun updatePlayerData(player: Player, position: Int)
    fun keepScreenOn(keepActive: Boolean)
}
